mastodon/Mastodon.py

```diff
@@ -818,46 +819,63 @@ def fetch_remaining(self, first_page):
     ###
     # Streaming
     ###
-    def user_stream(self, listener):
+    def user_stream(self, listener, async=False):
```

Comment:
the parameter name `async` is misleading here because the code is *always* async (processes via callbacks), the difference is just whether it happens in another thread or not.